eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/boot/EventMeshStartup.java

```diff
@@ -35,6 +36,7 @@ public static void main(String[] args) throws Exception {
             ConfigurationWrapper configurationWrapper =
                     new ConfigurationWrapper(EventMeshConstants.EVENTMESH_CONF_HOME,
                             EventMeshConstants.EVENTMESH_CONF_FILE, false);
+            ConfigurationContextUtil.setBaseConfigurationWrapper(configurationWrapper);
```

Comment:
Sorry, I am confused why you want to add this line. Could you please not modify eventmesh-runtime? Generally, eventmesh-runtime is not encouraged to be changed.

Reply:
Please refer to what eventmesh-connector-rocketmq looks like, so that we could merge your branch into master branch in the further future.

Reply:
> Sorry, I am confused why you want to add this line. Could you please not modify eventmesh-runtime? Generally, eventmesh-runtime is not encouraged to be changed.

Because we use the configuration center for unified management of configuration information, we hope to have a unified configuration reference, instead of piecemeal configuration like eventmesh-connector-rocketmq. It is really hard to understand from the perspective of redis-connector.  Therefore, I will remove this part of the function first, and then raise an issue to solve the configuration problem.